src/modules/decisions/components/results/ResultsContainer.tsx

```diff
@@ -39,6 +39,17 @@ const ResultsContainer = () => {
     cardWrapperStyles.justifyContent = 'space-between'
   }
 
+  const getRealResultsAmount = (searchState:any) => {
+    console.log(searchState);
```

Comment:
👀

Reply:
This is intentional for debugging purposes for now, just need to remember to remove it for the next real release.